### PR TITLE
Allow ansi-terminal 0.11

### LINKED ISSUE #200: Bump `ansi-terminal` to use 0.11
https://github.com/commercialhaskell/stackage/issues/5592

### PR DESCRIPTION
**Pull request by sorki**
Builds and passes test.

Closes #200.

### CODE REVIEW

File: co-log/co-log.cabal [hunk 1]
```diff
@@ -86,7 +86,7 @@ library
                            Colog.Pure
                            Colog.Rotation
 
-  build-depends:       ansi-terminal ^>= 0.10
+  build-depends:       ansi-terminal ^>= 0.11
```
Comment (vrom911) [inline co-log/co-log.cabal:89]: I would keep lower bound as well

```suggestion
  build-depends:       ansi-terminal >= 0.10 && < 0.12
```

**Review by vrom911 — COMMENTED** [review]
Thanks!

File: co-log/co-log.cabal (hunk 1 above)
Comment (sorki) [reply co-log/co-log.cabal]: Thanks, updated.

**Review by vrom911 — APPROVED** [review]
Thank you 👍🏼

**Pull request merged**